### ISSUE TITLE
Implement `lambda`-free schedulers

### ISSUE THREAD

**Issue by lucadiliello** [lucadiliello/transformers-lightning#43]
Schedulers used by the `huggingface` team use `lambda` functions to adjust the learning rate. However, `lambda` functions are not well digested by pickling algorithms and, in fact, `pytorch_lightning` refuses to start if a LambdaLR scheduler is passed within `configure_optimizers`.

Please implement the Schedulers of the `Transformers` library without lambda functions.
